DSP Accel Library/Testing and Images/testingTEMP.m

```diff
@@ -0,0 +1,52 @@
+
+%% Before Running
```

Comment:
This seems a little vague.  Is there another script that you have to run after this, but before you run the "After running" section?